### ISSUE TITLE
Replace documation order (slang/score)

### ISSUE THREAD

**Issue by meirwah** [openscore/openscore.github.io#5]
I think it will be wiser that the documation page will start with slang, and end with score.
WDYT?

**Comment by orius123** [comment]
adding @SamMarkowitz 
WDYT?

**Comment by SamMarkowitz** [comment]
From what I know so far, I think:

Keeping score first:
1. The project/site/etc are all called score, so it makes sense to start with that
2. Some sort of description of what score is should be towards the top

Moving slang to the beginning:
1. The score section my eventually grow to be pretty large and it will be long way before you hit something you can actually do - like a Hello World example

Bottom line: I think it's too early to tell (at least for me). We might have to wait to see how the docs grow. One possibility would be to add a fourth section (Getting Started, Overview, or whatever) that includes whatever we think should show up on top. (Also depending on how large the docs get, which I hope will not be too large, is to consider splitting the sections into different pages.)

**Comment by orius123** [comment]
currently not handling this until we figure out what we want to do.

**Comment by orius123** [comment]
@SamMarkowitz do you think we should keep this issue open?

**Comment by SamMarkowitz** [comment]
Yes, although I think it's not only about the order. It's more an overall organizational issue. The score section is almost empty and the developer section is over packed with developer and contributor information for both Slang and Score. The naming will affect this issue as well. Looks like we'll end up with one name, in which case we'll have to figure out how to name the sections, etc.

Bottom line. This needs more thought.

**Comment by orius123** [comment]
ok. so I'm closing this one and we will discuss it in general.
